RecoTracker/PixelLowPtUtilities/interface/StripSubClusterShapeTrajectoryFilter.h

```diff
@@ -27,7 +27,7 @@ namespace edm {
   class ConsumesCollector;
 }  // namespace edm
 
-//#define StripSubClusterShapeFilterBase_COUNTERS
+#define StripSubClusterShapeFilterBase_COUNTERS
```

Comment:
why is this uncommented?

Reply:
This is commented in the last commit. This was just used for testing purposes.